charts/stable/sonarr/templates/_exportarr.tpl

```diff
@@ -0,0 +1,32 @@
+{{/* Define the exportarr container */}}
+{{- define "sonarr.exportarr" -}}
+enabled: true
+type: Deployment
+podSpec:
+  containers:
+    exportarr:
+      enabled: true
+      primary: true
+      imageSelector: exportarrImage
+      args:
+        - sonarr
+      envFrom:
+        - configMapRef:
+            name: exportarr-config
+      probes:
+        readiness:
+          enabled: true
+          type: http
+          path: /healthz
+          port: {{ .Values.service.exportarr.ports.exportarr.port }}
+        liveness:
+          enabled: true
+          type: http
+          path: /healthz
+          port: {{ .Values.service.exportarr.ports.exportarr.port }}
+        startup:
+          enabled: true
+          type: http
+          path: /healthz
+          port: {{ .Values.service.exportarr.ports.exportarr.port }}
+{{- end -}}
```

Comment:
Please just do this in values.yaml.
No idea who told you this has to be seperate files tbh....